pkg/services/ngalert/models/alert_rule.go

```diff
@@ -113,6 +113,13 @@ type AlertRule struct {
 	Labels      map[string]string
 }
 
+type SchedulableAlertRule struct {
```

Comment:
Do you want to embed this in the full type that we fetch from the DB? Would it be easier to keep them up-to-date?

Reply:
In which case maybe it would be called:
"AlertRuleSchedulingData", "AlertRuleSchedulingModel"?

I have a preference to lead with "AlertRule" over "Schedule" because it scans more clearly to me.